src/gui/folderman.h

```diff
@@ -43,9 +43,14 @@ class LockWatcher;
 class TrayOverallStatusResult
 {
 public:
-    SyncResult::Status overallStatus;
-    bool hasUnresolvedConflicts;
     QDateTime lastSyncDone;
+
+    void addResult(const SyncResult &result);
+    void setStatus(SyncResult::Status status);
```

Comment:
Do we really need this setter? I think you could set the overall status within `addResult` by remembering the "worst" status you've seen so far, right?